vos/vos/vos.py

```diff
@@ -2737,3 +2777,291 @@ def get_job_status(self, url):
         """
         return VOFile(url, self.conn, method="GET",
                       follow_redirect=False).read()
+
+
+class Transfer(object):
```

Comment:
I would probably put this in its own module where all other common stuff (storage and vos) should go. utils.py or just 'transfer.py`?

Reply:
See technical debt TODO note in Transfer.get_transfer_error() w.r.t. VOFile.